Core/HLE/sceFont.cpp

```diff
@@ -645,9 +645,26 @@ static void __LoadInternalFonts() {
 	if (!pspFileSystem.GetFileInfo(fontPath).exists) {
 		pspFileSystem.MkDir(fontPath);
 	}
+	if (pspFileSystem.GetFileInfo("disc0:/PSP_GAME/USRDIR/zh_gb.pgf").exists) {
+		for (size_t i = 0; i < ARRAY_SIZE(fontRegistry); i++) {
```

Comment:
Well, I'm not sure it makes sense to affect ALL games for this one game with a patched font.prx.  Perhaps we could detect the patch somehow?

-[Unknown]